Fit-it-day placeholder update for Deploy to Production (Van) (#4895)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md

## Deploy a JavaScript app

> JavaScript apps built for production are a collection of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go.

Reviewer: ```suggestion
JavaScript apps built for production are a collection of static files. You can deploy them to any web server, and then configure that web server to be SPA-aware.
```

Reviewer: not sure if something like "and then you're ready to use them" is a reasonable replacement for "should be good to go" but if not I think it's fine to just cut. (not sure how well it translates)
